src/Tribe/Admin/Troubleshooting.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+/**
+ * Admin Troubleshooting for TEC plugins.
+ *
+ * @since TBD
+ *
+ * @package Tribe\Admin
+ */
+
+namespace Tribe\Admin;
+use \Tribe__Settings;
+use \Tribe__Main;
+use \Tribe__Admin__Helpers;
+use \Tribe__Timezones as Timezones;
+
+/**
+ * Class Admin Troubleshooting.
+ *
+ * @since TBD
+ *
+ * @package Tribe\Admin
+ */
+class Troubleshooting
+{
+    /**
+     * Slug of the WP admin menu item
+     *
+     * @since TBD
+     *
+     * @var string
+     */
+    const MENU_SLUG = 'tribe-troubleshooting';
```

Comment:
since this is new, let's go with `tec-troubleshooting` instead

Reply:
Cool, I've updated the slug.